scripts/tag_version.py

```diff
@@ -0,0 +1,19 @@
+import os
+import sys
+import re
+
+def main(version):
+    os.system("git checkout main && git pull")
+    os.system("git tag -a v_%s -m 'Version %s'" % (version, version))
```

Comment:
f"" string would look neater here maybe - but this works as well

Reply:
Yes, the fact is the Copilot wrote it and I was too lazy to change it ;P 
Let me refactor